.circleci/config.yml

```diff
@@ -190,74 +268,55 @@ jobs:
             done
           background: true
 
-      # Note: The setup.py checksum in the cache name ensures that we retrain our expert
-      # policies, which are cached in the pytest cache, whenever setup.py changes. This
-      # is just a rough heuristic to decide when the experts should be retrained but
-      # still better than nothing.
-      - restore_cache:
-          keys:
-            - v7-pytest-cache-{{ checksum "setup.py" }}
-            # This prefix-matched key restores the most recent pytest cache with any
-            # checksum. If that cached policy happens to be still loadable, we avoid
-            # retraining. See https://circleci.com/docs/2.0/caching/#restoring-cache
-            - v7-pytest-cache-
-          paths:
-            - .pytest_cache
+      - restore-pytest-cache
 
       - run:
           name: run tests
           command: |
             # Xdummy-entrypoint.py: starts an X server and sets DISPLAY, then runs wrapped command.
-            Xdummy-entrypoint.py pytest -n ${NUM_CPUS} --cov=/venv/lib/python3.8/site-packages/imitation \
+            Xdummy-entrypoint.py pytest -n ${NUM_CPUS} --cov=~/venv/lib/python3.8/site-packages/imitation \
                    --cov=tests --junitxml=/tmp/test-reports/junit.xml \
                     -vv tests/
             mv .coverage .coverage.imitation
             coverage combine  # rewrite paths from virtualenv to src/
 
       - codecov/upload
+      - save-pytest-cache
+      - store-test-output-unix
 
-      - save_cache:
-          key: v7-pytest-cache-{{ checksum "setup.py" }}
-          paths:
-            - .pytest_cache
+  unit-test-macos:
+    macos:
+      xcode: 13.4.1
 
-      - store_artifacts:
-          path: /tmp/test-reports
-          destination: test-reports
+    steps:
+      - dependencies-macos
+      - restore-pytest-cache
 
-      - store_test_results:
-          path: /tmp/test-reports
-          unit-test:
+      - run:
+          name: run tests
+          command: |
+            source ~/venv/bin/activate
+            # Xdummy-entrypoint.py: starts an X server and sets DISPLAY, then runs wrapped command.
+            ./ci/Xdummy-entrypoint.py pytest -n auto \
```

Comment:
Does Xdummy work on Mac? I thought the windowing system was different.

Also may not be necessary if CircleCI Mac executor isn't headless.

Reply:
I just copied this from the Linux tests. The `Xdummy-entrypoint.py` script doesn't crash or error on macOS, but I didn't check if it actually does what it's supposed to do. I'll just remove the usage of the script, and we can figure out X server stuff for macOS separately if we decide it's useful